ls_pre_commit_hooks/poetry-pytest.sh

```diff
@@ -1,7 +1,12 @@
-#!/bin/bash -e
+#!/bin/bash
+
+set -e
+set -o pipefail
+set -u
+set -x
```

Comment:
Why not this?

```suggestion
set -euxo pipefail
```

Reply:
I think it's equivalent. I personally prefer listing them explicitly.

It's a problem when you do that in the shebang: `#/bin/bash -euxo pipefail` and linters will complain about it:

https://github.com/koalaman/shellcheck/wiki/SC2096

So I just took the habit of writing them individually and then it's also easier to comment out just one flag temporarily for example, or enabling `set -x` only if `$DEBUG` is set and things like that.